client/app/scripts/superdesk-publish/publish.js

```diff
@@ -354,41 +365,67 @@
                     });
                 };
 
+                /**
+                 * Fetches list of global publish filters and returns the same.
+                 *
+                 * @return {*}
+                 */
                 var fetchGlobalPublishFilters = function() {
                     return filters.getGlobalPublishFilters().then(function(filters) {
                         $scope.globalFilters = filters;
                     });
                 };
 
+                /**
+                 * Fetch list of publish errors from the backend allowing the user to configure for the selected subscriber.
```

Comment:
Perhaps just split this into two lines, since it's a bit lengthy and requires horizontal scrolling.